src/packages/math-calculator/utils/calc.js

```
@@ -0,0 +1,308 @@
+  curState = STATE.INIT
+  curResult = 0
+  opNum1 = '0'
+  opNum2 = ''
+  op = ''
+}
+
+/**
+ * 是否为零
+ */
+function isZero(code) {
+  return code == '0'
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)